src/haproxy.py

```diff
@@ -25,7 +25,7 @@ def parse_args():
 def main():
     args = parse_args()
     template = args.prefix + '.{}.{}.{} {} ' + str(int(time()))
-    haproxy_info = read_ha_proxy_stats(args.haproxy_stats_socket)
+    haproxy_info = read_ha_proxy_stats(args.haproxy_stats_socket).decode('utf-8')
```

Comment:
I don't think this is unicode.  `.decode()` without argument should be sufficient.